platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -366,11 +303,14 @@ function ViewerViewportGrid(props) {
   const getViewportPanes = useCallback(() => {
     const viewportPanes = [];
 
-    const numViewports = viewportGridService.getNumViewportPanes();
-    for (let i = 0; i < numViewports; i++) {
+    for (let i = 0; i < viewports.length; i++) {
```

Comment:
I guess this was intentional for the auto populate feature @jbocce added

Reply:
Reverted back.